exercises/practice/robot-name/Cargo.toml

```diff
@@ -7,6 +7,9 @@ edition = "2024"
 # The full list of available libraries is here:
 # https://github.com/exercism/rust-test-runner/blob/main/local-registry/Cargo.toml
 [dependencies]
+# We use a wild-card specifier to minimize the risk of breaking existing
+# solutions in case the version in the test runner is updated.
+rand = "*"
```

Comment:
I'm not 100% sure about this. I'm investigating the possibility of keeping a given major version of a crate in the test runner forever. It should be possible, by giving old versions of a crate an alias. If we can provide support in the test runner for multiple major versions of a crate, I think we should declare a specific version here normally, without a wild-card.